app/views/shared/_analytics.html.erb

```diff
@@ -6,3 +6,13 @@
   gtag('js', new Date());
   gtag('config', '<%= @google_analytics_id %>');
 </script>
+
+<% if @google_tag_manager_id.present? %>
```

Comment:
I wonder if we should move the tag manager into its own layout, as to get to analytics we need to check if we have a `google_analytics_id` present: https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/views/layouts/base.html.erb#L4
or maybe we move the check for `google_analytics_id` inline here, so we keep the analytics in the same layout. Let me know your thoughts

Reply:
:+1: Much neater https://github.com/DFE-Digital/apply-for-teacher-training/pull/4488/commits/e167ade180a9c9495b30de4b508f33cc2aaaf3cf